src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -92,29 +91,6 @@ case class ValidateUserHasAccessToConsignment[T](argument: Argument[T]) extends
   }
 }
 
-object ValidateUserOwnsFiles extends AuthorisationTag {
```

Comment:
Assume a different AuthorisationTag is used for the new endpoints, and this is no longer needed as was only used by the old endpoints that are being deleted?

Reply:
Sort of. Before, we were creating the files and adding the metadata in a separate API call so we could check that the user owned the files but now we're creating the files and metadata in the same call so we can't and don't need to check any more. We're still checking the user owns the consignment on the new endpoint.